src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -479,6 +479,12 @@ bool crop_in_place_optimization::match(const program_node& node,
         // TODO: Need to allow optimization for gemm user
         if (node.is_dynamic() && (user->is_type<convolution>() || user->is_type<gemm>()))
             return false;
+        // If the user is strieded slice and the output shape is already static,
+        // the padding w/ dyn pad mask will not be propagated properly at runtime
+        if (user->is_type<strided_slice>()) {
+            if (node.is_dynamic() && !user->get_output_layout().is_dynamic())
+                return false;
```

Comment:
But if the user's output is static, isnt there same issue regardless it is strided slice or not?

Reply:
Agree. It doesn't seem to matter whether the user node is a specific type. The condition have been made more general.